_layouts/ontology_detail.html

```diff
@@ -17,17 +17,23 @@ <h1>
       <p>
         {{page.description}}
       </p>
+      {% if page.repository %}
+        {% if page.repository contains "https://github.com/" %}
+        <a href="{{ page.repository }}">
+          <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/{{ page.repository | remove: 'https://github.com/' }}" />
+          <img alt="GitHub contributors" src="https://img.shields.io/github/contributors/{{ page.repository | remove: 'https://github.com/' }}" />
+          <img alt="GitHub last commit" src="https://img.shields.io/github/last-commit/{{ page.repository | remove: 'https://github.com/' }}" />
+          <!--<img alt="GitHub issues" src="https://img.shields.io/github/issues/{{ page.repository | remove: 'https://github.com/' }}" />-->
+        </a>
+        {% endif %}
+      {% endif %}
       {% if page.twitter %}
-        <a href="https://twitter.com/{{ page.twitter }}" 
-           class="twitter-follow-button" 
-           data-show-count="false"
-           data-dnt="true">
-          Follow @{{ page.twitter }}
+        <a href="https://twitter.com/{{ page.twitter }}">
+          <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/{{ page.twitter }}?style=social">
         </a>
-        <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
       {% endif %}
       {% if page.google_plus %}
-      <a href="https://plus.google.com/{{page.google_plus}}">G+</a>
+        <a href="https://plus.google.com/{{page.google_plus}}">G+</a>
```

Comment:
Google plus does not exist anymore, I think this can be removed

Reply:
Done